homeassistant/components/coinbase/strings.json

```diff
@@ -38,5 +38,11 @@
       "currency_unavailable": "One or more of the requested currency balances is not provided by your Coinbase API.",
       "exchange_rate_unavailable": "One or more of the requested exchange rates is not provided by Coinbase."
     }
+  },
+  "issues": {
+    "removed_yaml": {
+      "title": "The Coinbase YAML configuration has been removed",
+      "description": "Configuring Coinbase using YAML has been removed.\n\nYour existing YAML configuration is not used by Home Assistant.\n\nRemove the YAML configuration from your configuration.yaml file and restart Home Assistant to fix this issue."
```

Comment:
There's still an import active. This text doesn't seem accurate.

Reply:
The import removal is in another PR https://github.com/home-assistant/core/pull/80084

Reply:
Then this PR should have been merged into that PR and it should also be marked as breaking change, not deprecation.

Reply:
I think it's better to adjust the text, and keep #80084 for the next release.

Reply:
@epenet Should not be needed, as it is an import from previous releases.

Reply:
Correct the text is probably non-ideal actually. The text should say deprecated not removed. My intention was the import would remain active for a further release then removed in the next release. I would suggest the new text should be:

"Configuring Coinbase using YAML has been deprecated.\n\nYour existing YAML configuration is not needed by Home Assistant.\n\nRemove the YAML configuration from your configuration.yaml file and restart Home Assistant to fix this issue."

Reply:
It should not be needed, the deprecation period has met the minimum required time. The added issue at this point, is nice, because it helps people to clean up their configuration.

Reply:
So we should go ahead with the current wording also merge #80084 with the deprecated schema then remove the schema in 2022.12?

Reply:
I have opened #81024 to fix the wording, as that was the original intention of @TomBrien